packages/twenty-server/src/modules/event/standard-objects/audit-log.object-metadata.ts

```diff
@@ -0,0 +1,81 @@
+import { FieldMetadataType } from 'src/engine/metadata-modules/field-metadata/field-metadata.entity';
+import { auditLogStandardFieldIds } from 'src/engine/workspace-manager/workspace-sync-metadata/constants/standard-field-ids';
+import { standardObjectIds } from 'src/engine/workspace-manager/workspace-sync-metadata/constants/standard-object-ids';
+import { FieldMetadata } from 'src/engine/workspace-manager/workspace-sync-metadata/decorators/field-metadata.decorator';
+import { IsNullable } from 'src/engine/workspace-manager/workspace-sync-metadata/decorators/is-nullable.decorator';
+import { IsSystem } from 'src/engine/workspace-manager/workspace-sync-metadata/decorators/is-system.decorator';
+import { ObjectMetadata } from 'src/engine/workspace-manager/workspace-sync-metadata/decorators/object-metadata.decorator';
+import { BaseObjectMetadata } from 'src/engine/workspace-manager/workspace-sync-metadata/standard-objects/base.object-metadata';
+import { WorkspaceMemberObjectMetadata } from 'src/modules/workspace-member/standard-objects/workspace-member.object-metadata';
+
+@ObjectMetadata({
+  standardId: standardObjectIds.auditLog,
+  namePlural: 'auditLogs',
+  labelSingular: 'Audit Log',
+  labelPlural: 'Audit Logs',
+  description: 'An audit log of actions performed in the system',
+  icon: 'IconIconTimelineEvent',
+})
+@IsSystem()
+export class AuditLogObjectMetadata extends BaseObjectMetadata {
+  @FieldMetadata({
+    standardId: auditLogStandardFieldIds.name,
+    type: FieldMetadataType.TEXT,
+    label: 'Event name',
+    description: 'Event name/type',
+    icon: 'IconAbc',
+  })
+  name: string;
+
+  @FieldMetadata({
+    standardId: auditLogStandardFieldIds.properties,
+    type: FieldMetadataType.RAW_JSON,
+    label: 'Event details',
+    description: 'Json value for event details',
+    icon: 'IconListDetails',
+  })
+  @IsNullable()
+  properties: JSON;
+
+  @FieldMetadata({
+    standardId: auditLogStandardFieldIds.context,
+    type: FieldMetadataType.RAW_JSON,
+    label: 'Event context',
+    description:
+      'Json object to provide context (user, device, workspace, etc.)',
+    icon: 'IconListDetails',
+  })
+  @IsNullable()
+  context: JSON;
+
+  @FieldMetadata({
+    standardId: auditLogStandardFieldIds.objectName,
+    type: FieldMetadataType.TEXT,
+    label: 'Object name',
+    description: 'If the event is related to a particular object',
+    icon: 'IconAbc',
+  })
+  @IsNullable()
```

Comment:
TEXT field shoud not be nullable (their default is '')

Reply:
If they are meant to be displayed yes, but if they aren't I feel like it makes more sense to have "null" vs '' (cf for example what you did for "key" on view table). Audit log being a very technical object, I don't think we should convert null to ''